Sources/LocationMessageCell.swift

```diff
@@ -69,11 +70,35 @@ open class LocationMessageCell: MessageCollectionViewCell<UIImageView> {
 
         let snapShotter = MKMapSnapshotter(options: snapshotOptions)
         snapShotter.start { (snapshot, error) in
-            self.activityIndicator.stopAnimating()
-            if error == nil {
-                self.messageContentView.image = snapshot?.image
+            defer {
+                self.activityIndicator.stopAnimating()
             }
+            guard let snapshot = snapshot, error == nil else {
+                print("\(#function) Error creating map snapshot: \(String(describing: error))") //show an error image?
```

Comment:
We don't want to `print` anything in production. Let's just return here

Reply:
Sure!, I'll fix it now